README

```diff
@@ -12,7 +12,7 @@ To build just the PDF:
 
 To build the whole tarball with docker images and all:
 
-	LICENSE=/path/to/gravwell-license make dist
+	VERSION=5.2.2 LICENSE=/path/to/gravwell-license make dist
```

Comment:
should this be 5.3.0 for the new release?

Reply:
yep